### PR TITLE
Fixes #17958: Create a configure to make the agent package -  configure target doesn't exist

### PR DESCRIPTION
**Pull request by peckpeck**
https://issues.rudder.io/issues/17958

### CODE REVIEW

**Comment by peckpeck** [comment]
PR updated with a new commit

File: rudder-agent/debian/rules [hunk 1]
```diff
@@ -103,12 +103,9 @@ ifeq ($(findstring systemd,$(WITH)),)
 DEPS := $(DEPS), systemd
 endif
 
-configure:
-	dh_testdir
-	cd SOURCES && ./configure $(WITH)
-
 build:
 	dh_testdir
+	cd SOURCES && ./configure $(WITH)
```
Comment (amousset) [inline rudder-agent/debian/rules:108]: why?

Comment (peckpeck) [reply rudder-agent/debian/rules:108]: why not ? :)

Comment (peckpeck) [reply rudder-agent/debian/rules:108]: That's the point of the parent PR : having a configure to call before calling make

Comment (amousset) [reply rudder-agent/debian/rules:108]: indeed :D but why moving the configure from the configure target to build?

Comment (peckpeck) [reply rudder-agent/debian/rules:108]: I don't know why the configre target was there first but it doesn't exist in debian specification and thus is never called

Comment (amousset) [reply rudder-agent/debian/rules:108]: :man_shrugging:

**Comment by peckpeck** [comment]
OK, squash merging this PR

**Pull request merged**